### ISSUE TITLE
After last Windows update, creating new entries in Toggl is completely broken

### ISSUE THREAD

**Issue by dulakm** [toggl/toggldesktop#2195]
Hello,

since last Windows app update, working with Toggl seems completely impossible. So far I have found out that:
- when adding new entry I cannot specify its description (on the video, I am hitting Enter key without a value submitting successfuly)
![ezgif com-crop1](https://user-images.githubusercontent.com/4852580/27051955-5a8d3560-4fb8-11e7-95e9-7f111a274755.gif)
- when modifying time started and then entry's description, time elapsed disappears
![ezgif com-crop2](https://user-images.githubusercontent.com/4852580/27051957-5c3597f4-4fb8-11e7-9c7b-56391ac04fb5.gif)
- application synchronizes itself while editing an entry and re-editing my entry while working on it
![animation 3](https://user-images.githubusercontent.com/4852580/27051975-677eb640-4fb8-11e7-9bee-3992dbfc99fa.gif)

Please, can you help me with the issues?

Thanks!

Regards,
Martin

**Comment by dulakm** [comment]
I have also tried to reinstall an older version but after reinstall, Toggl immediately updates itself. Please, how can I prevent Toggl from updating?

**Comment by TheSoloJake** [comment]
Case/486812

**Comment by ApocalypticFish** [comment]
/case/486048

**Comment by jlcanovas** [comment]
Same problem here (and in my office), we all have the same problem (Windows 7 & 10)

**Comment by tberta** [comment]
Same issues for me on Windows 7 in my office (with a proxy setup, that was running fine with previous versions). I'm using the latest Windows version : 7.4.38. 

The issues were present in a former version but I didn't take note of the version number. 
I updated to latest version 7.4.38 but the issues are still present.

**Comment by jlcanovas** [comment]
Seems to be fixed in [7.4.45 release](https://github.com/toggl/toggldesktop/releases/tag/v7.4.46).

**Comment by IndrekV** [comment]
We've released a new beta version that addresses the flickering issue. Please get the latest beta (7.4.47) by switching to beta channel from the About view

**Comment by IndrekV** [comment]
7.4.47 seems to be working ok. Pushed 7.4.47 to stable.

**Comment by dulakm** [comment]
Hello @IndrekV,

thank you for such a quick fix. I see most of the issues gone. However:
- adding new entries still misbehaves in way that there is always something fleshing (e.g. creating new empty entry for a second fills in information from old entry and then fleshes to a clear one)
- modyfing entry data persists but for a seconds fleshes with previous data,
- when creating new entry, the right panel isn't opening right away and I need to manually open it by clicking to entry description - the former things are simply weird, but this one really affects me as I have to do one redundant click.

![animation 5](https://user-images.githubusercontent.com/4852580/27172626-35b5e8fe-51b6-11e7-93ad-44bd7e71bb3a.gif)

Thank you,
Martin

**Comment by IndrekV** [comment]
@dulakm Thank you for all the details. We have reworked the syncing part to match the way it used to work before the glitches started happening. We've released a beta version with fixes v7.4.49. Switch to beta channel if you want to check out the fixes now or wait until next week when it will be pushed to stable.

As for the right panel not opening - This has always been so, but in macos there is a setting to enable opening edit view when user starts timer with a shortcut. We'll look into the possibility to add this also for windows.
